src/community/release/ext-importer-fgdb.xml

```diff
@@ -0,0 +1,19 @@
+<assembly>
+  <id>importer-fgdb</id>
```

Comment:
I have vague memories that a plugin module needs a name ending with -plugin or something bad (can't remember what) might happen.  At the very least all other community modules I've checked have an assembly whose name ends with -plugin.

Reply:
At a guess, it may affect the name of the zipfile created by the assembly process. I've updated it.